keylime/migrations/versions/eb869a77abd1_create_allowlist_table.py

```diff
@@ -0,0 +1,50 @@
+"""create_allowlist_table
+
+Revision ID: eb869a77abd1
+Revises: 8da20383f6e1
+Create Date: 2021-01-12 10:54:45.263268
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'eb869a77abd1'
+down_revision = '8da20383f6e1'
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+def upgrade_registrar():
+    pass
+
+
+def downgrade_registrar():
+    pass
+
+
+def upgrade_cloud_verifier():
+    op.create_table('allowlists',
+                    sa.Column('id', sa.Integer(), nullable=False),
+                    sa.Column('name', sa.String(255), nullable=False),
+                    sa.Column('tpm_policy', sa.Text(), nullable=True),
+                    sa.Column('vtpm_policy', sa.Text(), nullable=True),
+                    sa.Column('allowlist', sa.Text(length=429400000), nullable=True),
+                    sa.PrimaryKeyConstraint('id'),
+                    sa.UniqueConstraint('name', name='uniq_allowlists0name'),
+                    mysql_ENGINE='InnoDB',
+                    mysql_DEFAULT_CHARSET='UTF8'
+                    )
+
+
+def downgrade_cloud_verifier():
+    pass
```

Comment:
Shouldn't the downgrade remove the `allowlists` table?

Reply:
Technically, yes, but in practice it's not used that much, we normally ignore downgrade implementation :)
Can do if desired.

Reply:
You're right that it's not used much in practice (fail forward is a much better model than downgrading) but we do provide it for the other migrations if I'm not mistaken. So we should at least be consistent. I would be fine if we decided to skip downgrading all together and remove that from the other migrations as well. But that would be a separate issue.

So I'll let you pick which you prefer: add downgrade code to this migration or add a ticket to remove the downgrade from the other migrations.